context/app/static/js/components/Detail/ProvTable/ProvTable.jsx

```diff
@@ -34,34 +22,28 @@ function ProvTable(props) {
   return (
     <FlexContainer>
       {types.map((type, i) => (
-        <React.Fragment key={`provenance-list-${typesToSplit[i].toLowerCase()}`}>
-          <ListColumn
-            subheader={
-              <CenteredListSubheader component="div" color="primary">
-                {typesToSplit[i]}
-              </CenteredListSubheader>
-            }
-          >
-            <Divider />
+        <TableColumn key={`provenance-list-${typesToSplit[i].toLowerCase()}`}>
+          <EntityColumnTitle variant="h5">{typesToSplit[i]}s</EntityColumnTitle>
+          <FlexColumn>
             {type && type.length ? (
-              type.map((item) => (
-                <ListItemLink
-                  key={item['hubmap:uuid']}
-                  href={`/browse/dataset/${item['hubmap:uuid']}`}
-                  disabled={uuid === item['hubmap:uuid']}
-                >
-                  <ListItemText primary={item['hubmap:displayDOI']} />
-                </ListItemLink>
+              type.map((item, j) => (
+                <ProvTableTile
+                  key={item.uuid}
+                  uuid={item.uuid}
+                  id={item.display_doi}
+                  entity_type={item.entity_type}
+                  isCurrentEntity={uuid === item.uuid}
+                  isNotSibling={j > 0 ? type[j - 1].specimen_type !== item.specimen_type : false}
```

Comment:
Ternary not needed?
```suggestion
                  isNotSibling={j > 0 && type[j - 1].specimen_type !== item.specimen_type}
```
... and prefer not to have negative variables?
```suggestion
                  isSibling={!(j > 0 && type[j - 1].specimen_type !== item.specimen_type)}
```
... and then simplify?
```suggestion
                  isSibling={j === 0 || type[j - 1].specimen_type === item.specimen_type)}
```

Not a blocker for merge.